### PR TITLE
feat(aerial): Config imagery north-island-cyclone-gabrielle_2023_10m into Aerial Map. 

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: config/tileset/aerial.json [hunk 1]
```diff
@@ -100,6 +100,14 @@
       "title": "Snares Islands/Tini Heke 0.5m Satellite Imagery (2019-2020)",
       "category": "Satellite Imagery"
     },
+    {
+      "2193": "s3://linz-basemaps/2193/north-island-cyclone-gabrielle_2023_10m/01GT86B8BWT1WQEAQK7CE4KEXA",
+      "3857": "s3://linz-basemaps/3857/north-island-cyclone-gabrielle_2023_10m/01GT86BYJ4VBD14QKA0HN76FE2",
+      "name": "north-island-cyclone-gabrielle_2023_10m",
+      "title": "North-island-cyclone-gabrielle 2023 10m",
+      "minZoom": 5,
```
Comment (Wentao-Kuang) [inline config/tileset/aerial.json:108]: We might want to update the minZoom to 32.

Comment (rclarkelinz) [inline config/tileset/aerial.json:108]:
```suggestion
      "minZoom": 32,
```

File: config/tileset/aerial.json [hunk 2]
```diff
@@ -100,6 +100,14 @@
...
+      "category": "Satellite Imagery"
```
Comment (rclarkelinz) [inline config/tileset/aerial.json:109]:
```suggestion
      "category": "Event"
```

**Review by rclarkelinz — APPROVED** [review]

File: config/tileset/aerial.json [hunk 3]
```diff
@@ -100,6 +100,14 @@
       "title": "Snares Islands/Tini Heke 0.5m Satellite Imagery (2019-2020)",
       "category": "Satellite Imagery"
     },
+    {
+      "2193": "s3://linz-basemaps/2193/north-island-cyclone-gabrielle_2023_10m/01GT86B8BWT1WQEAQK7CE4KEXA",
+      "3857": "s3://linz-basemaps/3857/north-island-cyclone-gabrielle_2023_10m/01GT86BYJ4VBD14QKA0HN76FE2",
+      "name": "north-island-cyclone-gabrielle_2023_10m",
+      "title": "North-island-cyclone-gabrielle 2023 10m",
```
Comment (dwsilk) [inline config/tileset/aerial.json:107]:
```suggestion
      "title": "North Island 10m Cyclone Gabrielle Satellite Imagery (2023)",
```

**Review by dwsilk — CHANGES_REQUESTED** [review]

**Review by dwsilk — APPROVED** [review]

**Pull request merged**